crates/forge_main/src/ui.rs

```diff
@@ -73,6 +73,31 @@ impl<F: API> UI<F> {
         }
     }
 
+    async fn get_update_configuration(&self) -> Result<UpdateConfiguration> {
+        let mut workflow = Workflow::default();
+        let user_workflow = self.api.read_workflow(&self.workflow_path()).await?;
+        workflow.merge(user_workflow);
+
+        Ok(UpdateConfiguration {
+            check_frequency: workflow
+                .updates
+                .get("check_frequency")
+                .cloned()
+                .unwrap()
+                .as_str()
+                .unwrap()
+                .to_string()
+                .into(),
+            auto_update: workflow
+                .updates
+                .get("auto_update")
+                .cloned()
+                .unwrap()
+                .as_bool()
+                .unwrap(),
+        })
+    }
```

Comment:
use `read_workflow` to get this information.

Reply:
Done